ionoscloud/resource_volume.go

```diff
@@ -65,6 +65,11 @@ func resourceVolume() *schema.Resource {
 				Elem:     &schema.Schema{Type: schema.TypeString},
 				Optional: true,
 			},
+			"ssh_keys": {
```

Comment:
@gabisavu-ionos please add some tests in `resource_volume_test.go` that reflect the changes made in `resource_volume.go`.

Reply:
I will mark this as solved since we will remove `ssh_keys` and `ssh_key_path` for the volume resource. If we were to add tests now, it would mean deleting them shortly after, so it' better not to add them at all.